src/Microsoft.Identity.Client/PlatformsCommon/Shared/AbstractPlatformProxy.cs

```diff
@@ -157,5 +157,10 @@ public string GetProductName()
         protected abstract string InternalGetProductName();
         protected abstract ICryptographyManager InternalGetCryptographyManager();
         protected abstract IPlatformLogger InternalGetPlatformLogger();
+
+        public virtual ITokenCacheBlobStorage CreateTokenCacheBlobStorage() 
+        {
+            return new NullTokenCacheBlobStorage();
```

Comment:
>NullTokenCacheBlobStorage [](start = 23, length = 25)

lazy loaded process static (as in other places)?

Reply:
I disagree. The "CreateX" methods new up objects. The GetX methods can returned cached values. CreateTokenCacheBlobStorage is similar to CreateTokenCacheAccessor and CreateLegacyCachePersistence. We shouldn't return the same object in all the TokenCache instances as it introduces side effects.

Reply:
Why is that a benefit with NullTokenCacheBlobStorage? It's supposed to consistently do nothing. What you gaining by newing it up every time?